### ISSUE TITLE
[suggestion] ROS on conda-forge

### ISSUE THREAD

**Issue by jf---** [compas-dev/compas_fab#85]
not every day you hear of advances that get you in full tribal dance mode, but I'd say this is a candidate

[ROS on conda-forge](https://medium.com/@wolfv/ros-on-conda-forge-dca6827ac4b6)

**Comment by romanarust** [comment]
OMG, awesome!

**Comment by xarthurx** [comment]
No windows yet~

**Comment by jf---** [comment]
> No windows yet~

Interestingly enough the author mentions working together with [Sean Yen](https://twitter.com/sean_yen), of MS, working on ROS

> "This makes me confident that with his deep knowledge of Windows & ROS, we’ll be able to support Windows for the ROS melodic core packages on conda soon"

Not bad eh ;)

**Comment by gonzalocasas** [comment]
I can confirm this actually works, at least with ROS Noetic installing via conda on Windows 10! 🎉

**Comment by xarthurx** [comment]
@gonzalocasas  You mean using ROS without WSL?

**Comment by gonzalocasas** [comment]
@xarthurx yes! the whole pipeline works now, ros/moveit/rviz, everything running natively on Windows:
![image](https://user-images.githubusercontent.com/933277/112855224-e7a52080-90ae-11eb-817e-125ceea789d2.png)

**Comment by jf---** [comment]
@gonzalocasas any thoughts on what bar the conda ROS install would need to meet, to be considered a viable alternative to docker?

**Comment by gonzalocasas** [comment]
@jf--- actually, it is already very viable. Internally, some projects have been running with that for a couple of months now, we will add docs for it to the main documentation in the short term! ;)

**Comment by gonzalocasas** [comment]
I realize that I already said that before, but now we've tested it and are sure it works, I was aiming at adding it to the docs these sprint, but perhaps it slips to the next one in a week or two.

**Comment by Tobias-Fischer** [comment]
Hiya - just came across this issue by chance. @wolfv, @traversaro and I would be more than happy to help out if you encounter any issues in running RoboStack, but it seems like things are running smoothly already which is great :)
